### PR TITLE
chore: update vue api doc to match implementation

### PR DESCRIPTION
**Pull request by davidnixon**
Fix for https://github.com/testing-library/vue-testing-library/issues/280

### CODE REVIEW

**Review by timdeschryver — APPROVED** [review]

File: docs/vue-testing-library/api.mdx [hunk 1]
```diff
@@ -209,12 +183,15 @@ An alias for `@vue/test-utils`
...
+#### `rerender(props)`
 
 An alias for `@vue/test-utils`
 [setProps](https://vue-test-utils.vuejs.org/api/wrapper/#setprops).
 
-It returns a Promise through `wait()`, so you can `await updateProps(...)`.
+It returns a Promise through so you can `await rerender(...)`.
+
+See
+[update properties example](vue-testing-library/examples.mdx#example-updating-properties).
```
Comment (afontcu) [inline docs/vue-testing-library/api.mdx:194]: see comment below

```suggestion
```

File: docs/vue-testing-library/api.mdx [hunk 2]
```diff
@@ -164,13 +147,10 @@ renders its HTML directly in the body.
 > Note: the queries returned by the `render` looks into `baseElement`, so you
 > can use queries to test your portal component without the `baseElement`.
 
-#### `debug(element)`
```
Comment (afontcu) [inline docs/vue-testing-library/api.mdx:167]: debug still accepts an element:

https://github.com/testing-library/vue-testing-library/blob/main/src/render.js#L42

File: docs/vue-testing-library/api.mdx [hunk 3]
```diff
@@ -7,29 +7,27 @@ title: API
...
   - [`update(elem, value)`](#updateelem-value)
 - [`cleanup`](#cleanup)
 
 ---
 
-## `render(Component, options, callback)`
+## `render(Component, options)`
 
 The `render` function is the only way of rendering components in Vue Testing
 Library.
```
Comment (afontcu) [inline docs/vue-testing-library/api.mdx:33]: there's a sentence below this one that says:

"It takes up to 3 parameters and returns an object with some helper methods."

it should be

"It takes up to 2 parameters and returns an object with some helper methods."

**Review by afontcu — CHANGES_REQUESTED** [review]
thank you for this!! 🙌  I left some comments here and there

**Review by afontcu — APPROVED** [review]
thank you! 🤩

**Comment by afontcu** [comment]
@all-contributors please add @davidnixon for docs

**Pull request merged**